### ISSUE TITLE
No issue, just wondering how to view backend log

### ISSUE THREAD

**Issue by peterkuli** [xirixiz/dsmr-reader-docker#316]
### Request

I use DSMR with Docker, obviously :)

I'm getting messages that the backup failed and that I have to look in the backend log. Now, how do I find this? I think this is more a noob Docker question, but I've looked in the issues here but I cannot find anywhere where the log file is and how to navigate to it, since it's in the Docker image.

I'm using the following run command:

`docker run -d \
  --name dsmr \
  --restart always \
  -p 7777:80 \
  -p 7779:443 \
  -e DJANGO_TIME_ZONE=Europe/Amsterdam \
  -e DJANGO_DATABASE_HOST=192.168.1.11 \
  -e DJANGO_DATABASE_USER=dsmrreader \
  -e DJANGO_DATABASE_PASSWORD=dsmrreader \
  -e VIRTUAL_HOST=localhost \
  --no-healthcheck \
  --device /dev/ttyUSB0:/dev/ttyUSB0 \
  xirixiz/dsmr-reader-docker`

So basically the question is, how and where can I view the log file?

Update: when I use this command to go into the container: `docker exec -t -i dsmr /bin/bash`
Then I only see a `nginx` folder inside `/var/log`

Also, I don't have a `supervisord.log` inside `/tmp`

**Comment by xirixiz** [comment]
Hehehe, no worries. Hit ```docker logs -f <container name>```

It seems your container name is ```dsmr```

**Comment by peterkuli** [comment]
@xirixiz Thanks! That worked for me! Now let's hope I can resolve the issues ;-)
